### DISCUSSION TITLE
Help -  AIRSDK Archives??

### DISCUSSION THREAD

**Discussion by nc35** [airsdk/Adobe-Runtime-Support#2044] [General]
Hi Everyone,
Bit of a noob so please be kind.
Where can I find archives for the AirSDK?  Looking for 30.0 or older, please.
For the life of me I cannot get 33.1 to do what I want when compiling to an android .apk
30.0 worked perfectly.
Thanking you all in anticipation.

**Comment by ajwfrost** [comment]
The Adobe SDK packages aren't available any more I'm afraid - and they wouldn't help you if you needed to package for Android Play Store as the requirements have changed since then...

What are you having problems with in the 33.1 build tools? Have you tried installing Android Studio, since this would help getting everything set up for the new build system..

thanks

**Comment by nc35** [comment]
Hi Andrew,
Thank you for your reply.
I did manage to find a 3rd party archive, but as you said no good as not compatible.
Now trying to re write the code.
Thanks again for your reply.
